apps/ledger-live-desktop/src/main/updater/sslHelper.ts

```diff
@@ -1,13 +1,76 @@
 import crypto from "crypto";
-import secp256k1 from "secp256k1";
+import * as ecc from "@bitcoinerlab/secp256k1";
```

Comment:
This is just a wrapper on top of `@noble/curves`, but with... older API and decisions like linked above

If you use `@noble/curves` directly, you won't need `asn1.js` dep, as noble supports der.